packages/react-composites/src/composites/CallComposite/hooks/useAdaptedSelector.ts

```diff
@@ -68,7 +62,7 @@ export const useSelectorWithAdaptation = <
       if (!mounted.current) {
         return;
       }
-      const newProps = selector(adaptState(state), selectorProps ?? callConfigProps);
+      const newProps = selector(adaptState(state), selectorProps ?? { callId: state.call?.id });
```

Comment:
I assume it's ok `{ callId: state.call?.id }` will be a new object each time `selector` is called, but perhaps we should check that `selector` from reselect doesn't require the props to be memoized

Reply:
Looking at their docs for create selector it does not seem that they need anything to be memoized. they have lots of options to memoize the selector though
https://reselect.js.org/api/createSelector

Reply:
Just verified this, put a console log inside the `if (propRef.current !== newProps) {` (and one in an `else` to that) and validate there is no difference if a cached `{ callId }` is used vs creating the callId object each pass 👍